### PR TITLE
[TECNICAL SUPPORT] LPS-87075 Only add user tracker if node is master

### PR DESCRIPTION
**Pull request by amadeafejes**
Hi Eric,

I'm not sure if you are the right person to review this cluster related pull request. If yes, could you please review it in other case could please provide me the appropriate person?

Thank you in advance,
Amadea

Related LPS:
https://issues.liferay.com/browse/LPS-87075

### CODE REVIEW

**Comment by amadeafejes** [comment]
ci:test:relevant

**Comment by amadeafejes** [comment]
ci:test:relevant

**Comment by ericyanLr** [comment]
Hi @amadeafejes ,

I'm actually not sure which team owns the Live Users / User Tracking feature.
But, after taking a look at the pr, here are my thoughts:

**Currently, it is unclear whether live users should be tracked by every node in the cluster**
A reason why this may be wanted is to be able to view a list of all users throughout the cluster.
- e.g. Viewing a list of users under Control Panel -> Users -> Monitoring

Reference: https://community.liferay.com/forums/-/message_boards/message/78279783

**Scenario: If it is expected that each node is responsible for tracking it's own live users**
If this is the case, the current proposed fix does not account for the following:

- Tracking users only if the user is on the current node (the proposed fix matches the master node)
- The proposed fix does not account for signIn. It only accounts for signOut.

----
To summarize: It would be great if we can find out whether the current behavior is intentional, where each node is receiving information about a live user and attempting to track the user.

I think it's worth a try to reach out to the User Management SME to verify if they have ownership on this feature, since **Monitoring** is under **Control Panel -> Users**.

If you know that the scenario above is the expected behavior, please let me know your thoughts regarding my remarks on the current pr.

Closing this pr for now.

Please let me know if you have any additional questions.
Thanks!

**Comment by amadeafejes** [comment]
Hi Eric,

Thank you for checking the pull. Currently I can't give you more usable information, but I asked Christopher Kian, the User Management SME. See the [associated SME request ticket](https://issues.liferay.com/browse/LPP-31934?focusedCommentId=1620262&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1620262).

Br,
Amadea
